openupgrade_scripts/scripts/account_peppol/18.0.1.1/upgrade_analysis.txt

```diff
@@ -10,6 +10,7 @@ account_peppol / res.partner              / account_peppol_is_endpoint_valid (bo
 account_peppol / res.partner              / account_peppol_validity_last_check (date): DEL
 account_peppol / res.partner              / invoice_sending_method (False): NEW selection_keys: ['email', 'manual', 'peppol', 'snailmail'], mode: modify
 account_peppol / res.partner              / is_peppol_edi_format (boolean): module is now 'account_edi_ubl_cii' ('account_peppol')
+account_peppol / res.partner              / peppol_eas (False)            : NEW selection_keys: ['0002', '0007', '0009', '0037', '0060', '0088', '0096', '0097', '0106', '0130', '0135', '0142', '0151', '0183', '0184', '0188', '0190', '0191', '0192', '0193', '0195', '0196', '0198', '0199', '0200', '0201', '0202', '0204', '0208', '0209', '0210', '0211', '0213', '0216', '0218', '0221', '0225', '0230', '0235', '0240', '9910', '9913', '9914', '9915', '9918', '9919', '9920', '9922', '9923', '9924', '9925', '9926', '9927', '9928', '9929', '9930', '9931', '9932', '9933', '9934', '9935', '9936', '9937', '9938', '9939', '9940', '9941', '9942', '9943', '9944', '9945', '9946', '9947', '9948', '9949', '9950', '9951', '9952', '9953', '9955', '9957', '9959', 'AN', 'AQ', 'AS', 'AU', 'EM', 'odemo'], mode: modify
```

Comment:
Field was previously declared only in `account_edi_ubl_cii`. Now they declared in `account_peppol` too. See https://github.com/odoo/odoo/commit/76cd731a0c5c8990285d2abaf99226fc70d993bd. It should be NOTHING TO DO, also `account_peppol` is not migrated yet.